viewer/src/main/webapp/META-INF/context.xml

```diff
@@ -3,7 +3,7 @@
   <Parameter name="componentregistry.path" override="false" value="/viewer-html/components"/>
   <Parameter name="flamingo.data.dir" override="false" value="/opt/tailormap_data"/>
   <Parameter name="flamingo.solr.url" override="false" value="https://tailormap.nl/solr/"/>
-  <Parameter name="flamingo.restproxy.port" override="false" value="8084"/>
+  <Parameter name="flamingo.restproxy.port" override="false" value="8080"/>
```

Comment:
Niet committen

Reply:
Die er door geglipt, moet even zoeken hoe ik die uit de PR haal, of misschien draai ik de wijziging ook wel terug, volgens mij heeft deze wijziging toch geen effect gehad.

Reply:
Heb ik terugveranderd naar 8084, opnieuw build all gedaan en dan werkt het nog steeds. Wordt mogelijk niet gebruikt?